src/components/cursor.js

```diff
@@ -275,7 +275,11 @@ module.exports.Component = registerComponent('cursor', {
     }
 
     // Unset current intersection.
-    if (this.intersectedEl) { this.clearCurrentIntersection(); }
+    if (this.intersectedEl) {
```

Comment:
I always wondered if this check should be inside the `clearCurrentIntersection` method. It should probably check and return immediately.